src/backend/tests/unmocked/team-type.test.ts

```diff
@@ -42,4 +42,17 @@ describe('Team Type Tests', () => {
       expect(result).toStrictEqual([teamType1, teamType2]);
     });
   });
+
+  describe('Get a single team type', () => {
+    it('Get a single team type works', async () => {
+      const teamType1 = await TeamsService.createTeamType(superman, 'teamType1', 'YouTubeIcon');
+      const result = await TeamsService.getSingleTeamType(teamType1.teamTypeId);
+      expect(result).toStrictEqual(teamType1);
+    });
+
+    it('Get a single team type fails', async () => {
+      const nonExistingTeamTypeId = 'nonExistingId';
+      await expect(async () => TeamsService.getSingleTeamType(nonExistingTeamTypeId)).rejects.toThrow(NotFoundException);
```

Comment:
Throw a `new NotFoundException` and make sure it has the exact description of the exception that is being thrown. Look above at other tests that throw exceptions for an example